site/docs/uz/advanced/structuring.md

```diff
@@ -0,0 +1,68 @@
+---
+prev: ./middleware.md
+next: ./scaling.md
+---
+
+# Kengaytirish I: Katta kod bazasi
+
+Sizning botingiz murakkablashib borishi bilan siz dastur kodlari bazasini qanday tuzish masalasiga duch kelasiz.
+Tabiiyki, siz uni fayllarga bo'lishingiz mumkin.
+
+## Yechim
+
+> grammY hali juda yosh va hali DI konteynerlari bilan rasmiy integratsiyani ta'minlamaydi.
+> Biz buni qo'llab-quvvatlasak, xabardor bo'lish uchun [@grammyjs_news](https://t.me/grammyjs_news) ga obuna bo'ling.
+
+Siz o'zingizning kodingizni xohlaganingizcha tuzishingiz mumkin va barchaga mos keladigan yagona yechim yo'q.
+Aytish joizki, kodingizni tuzishning sodda va tasdiqlangan strategiyasi quyidagilardir.
+
+1. Semantik jihatdan bir xil faylda (yoki kod hajmiga, jildga qarab) bir-biriga tegishli narsalarni birlashtiring.
```

Comment:
remove "bir xil faylda"

Reply:
tegishli => mos keladigan

Reply:
birlashtiring => bir faylda birlashtiring

Reply:
Useful hint for you guys: you can click the suggestion formatting button to add inline suggestions:
```suggestion
1. Semantik jihatdan bir xil faylda (yoki kod hajmiga, jildga qarab) bir-biriga tegishli narsalarni birlashtiring. Yay!
```
(Don't apply this)

These suggestions can directly be committed from the web interface. However, if you go to the Files tab, you can add several suggestions to one batch and commit them all at once. That way, you don't end up with hundreds of commits :)

Reply:
Thanks for useful tip :)